### ISSUE TITLE
restore crnk-example-frontend

### ISSUE THREAD

**Issue by JonasStn** [crnk-project/crnk-example#19]
the crnk-example-frontend folder was deleted on master. Was it on purpose? It's confusing when one tries to find the examples for angular implementation.

**Comment by remmeier** [comment]
Hi. well if it is desired, it can be added again. I noticed there are still some traces left. But it would also have to get an update in various areas.

**Comment by remmeier** [comment]
new plan in the area :-) crnk-ui will get a significant upgrade and show the various techniques in there. This way only one UI has to be maintained and we will show advanced sorting, filtering, facetted search,...

**Comment by goldyliang** [comment]
Does it mean that the crnk-ui is already supreme to the angular frontend application, or it is just desired to promote crnk-ui?  
This page still has a reference to the angular frontend though, and I suggest you to remove it if that is desired: https://www.crnk.io/examples/

**Comment by remmeier** [comment]
no, not at all currently. Quite the contrary actually, it still needs some work... 

Reason for dropping the frontend example are two-fold: there are many different frameworks out there, each evolving quite rapidly. So it is both maintenance effort and rather impossible to cover a wide spectrum of frameworks. On the otherside, crnk/jsonapi is plain, simple REST, and not something more complex as OData, GraphQL and similar where you also need client tooling to work with it. 

but contributions are welcomed if somebody would like to maintain one or help getting crnk-ui a bit more up to date.
